src/react-apps/applications/ux-editor/src/containers/DroppableDraggableContainer.tsx

```diff
@@ -108,6 +105,48 @@ const dropTargetSpec: DropTargetSpec<IDroppableDraggableContainerProps> = {
       }
     }
   },
+  hover(props: IDroppableDraggableContainerProps, monitor: DropTargetMonitor, component: any) {
```

Comment:
Is there a reason for using any for the component here? Otherwise explicitly set the type.

Reply:
lets keep it as any until we fix containers.